webpack.config.js

```diff
@@ -59,9 +69,27 @@ const serverConfig = merge.smart(
     },
     platformConfig('server'),
 );
+const pagesConfig = merge.smart(
```

Comment:
not too sure what's happening here?

Reply:
This will generate a bundle for each page. The server requires each bundle under the dist/pages directory using requireDir

Reply:
it looks like i tried that at some point! https://github.com/guardian/guui/blob/master/.babelrc.js#L43 

i sounds like a good idea to me?